sig-list.md

```diff
@@ -31,11 +31,11 @@ Each group's material is in its subdirectory in this project.
 | Name | Label [1] | Chairs [2] | Contact | Meetings |
 |------|-----------|---------|---------|----------|
 |[sig-compute](sig-compute/charter.md) |[sig/compute](https://github.com/kubevirt/kubevirt/labels/sig/compute)|<ol><li>[Itamar Holder](https://github.com/iholder101), Red Hat</li><li>[Jed Lejosne](https://github.com/jean-edouard), Red Hat</li></ol> |[Slack](kubevirt-dev)<br/> [Mailing List](https://groups.google.com/forum/#!forum/kubevirt-dev) |<ul></ul> |
-|[documentation](sig-documentation/charter.md) |[](https://github.com/kubevirt/kubevirt/labels/)| | |<ul></ul> |
-|[storage](sig-storage/charter.md) |[](https://github.com/kubevirt/kubevirt/labels/)| | |<ul></ul> |
-|[testing](sig-testing/charter.md) |[](https://github.com/kubevirt/kubevirt/labels/)| | |<ul></ul> |
-|[network](sig-network/charter.md) |[](https://github.com/kubevirt/kubevirt/labels/)| | |<ul></ul> |
-|[KubeVirt Perf and Scale SIG](sig-scale/charter.md) |[kubevirt-community](https://github.com/kubevirt/kubevirt/labels/kubevirt-community)|<ol><li>[Ryan Hallisey](https://github.com/rthallisey), Nvidia</li><li>[Alay Patel](https://github.com/alaypatel07), Nvidia</li></ol> |[Slack](kubevirt-dev)<br/> [Mailing List](https://groups.google.com/forum/#!forum/kubevirt-dev) |<ul><li>KubeVirt Perf and Scale SIG Meeting: [ Thursday at 14:00 UTC (bimonthy) ](https://zoom.us/j/3205945033)</li></ul> |
+|[documentation](sig-documentation/charter.md) |[sig/documentation](https://github.com/kubevirt/kubevirt/labels/sig/documentation)| | |<ul></ul> |
+|[storage](sig-storage/charter.md) |[sig/storage](https://github.com/kubevirt/kubevirt/labels/sig/storage)| | |<ul></ul> |
+|[testing](sig-testing/charter.md) |[sig/testing](https://github.com/kubevirt/kubevirt/labels/sig/testing)| | |<ul></ul> |
+|[network](sig-network/charter.md) |[sig/network](https://github.com/kubevirt/kubevirt/labels/sig/network)| | |<ul></ul> |
+|[KubeVirt Perf and Scale SIG](sig-scale/charter.md) |[](https://github.com/kubevirt/kubevirt/labels/)|<ol><li>[Ryan Hallisey](https://github.com/rthallisey), Nvidia</li><li>[Alay Patel](https://github.com/alaypatel07), Nvidia</li></ol> |[Slack](kubevirt-dev)<br/> [Mailing List](https://groups.google.com/forum/#!forum/kubevirt-dev) |<ul><li>KubeVirt Perf and Scale SIG Meeting: [ Thursday at 14:00 UTC (bimonthy) ](https://zoom.us/j/3205945033)</li></ul> |
```

Comment:
Did we want to include the sig-scale label?
https://github.com/kubevirt/kubevirt/labels?q=scale

Reply:
Oh, good point, thanks for your eagle-eyes :)